sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -47,15 +48,23 @@ object CommandUtils extends Logging {
     }
   }
 
-  def calculateTotalSize(sessionState: SessionState, catalogTable: CatalogTable): BigInt = {
+  def calculateTotalSize(spark: SparkSession, catalogTable: CatalogTable): BigInt = {
+    val sessionState = spark.sessionState
     if (catalogTable.partitionColumnNames.isEmpty) {
       calculateLocationSize(sessionState, catalogTable.identifier, catalogTable.storage.locationUri)
```

Comment:
It sounds like we are not following the same file filtering for non-partition tables.

Reply:
This filtering of temp files is not done in other places where calculateLocationSize is called directly (eg AlterTableAddPartition). Should we skip checking for the temp files since they don't make a lot of difference to the total size?

Reply:
If not, we have to make changes in all those places too.

Reply:
Let us make them consistent. only count the data files.

Reply:
Got it, maybe we can add the filter inside calculateLocationSize. Let me know if that's okay, I can make the changes.

Reply:
I have made the changes for this in calculateLocationSize.